book/getting-started/linux-setup.md

```diff
@@ -208,9 +208,8 @@ cd ~/wallaroo-tutorial
 This will be our base directory in what follows. If you haven't already cloned the Wallaroo repo, do so now (this will create a subdirectory called `wallaroo`):
 
 ```bash
-git clone https://github.com/WallarooLabs/wallaroo
+git clone --depth 1 -b release-0.1.2 https://github.com/WallarooLabs/wallaroo
```

Comment:
will -b work when its a tag? i assume yes. want to verify.

Reply:
To be clear.

Manual versioning that I have to do means replacing the name of the branch `release-0.1.2` with the tag `0.1.2` on release. So whatever we have needs to easily work with that. 

Wget for a release doesn't. If this requires more to change when going from branch to tag, I think we should hold off on this until we have a more sustainable release process that doesn't involve me manually changing some values.